client/plots/genomeBrowser.js

```diff
@@ -352,6 +374,25 @@ class genomeBrowser {
 			this.blockInstance.tk_remove(tkidx)
 		}
 	}
+
+	async maySaveMds3SubtkToState(bb) {
+		/* arg is block instance
+		when a mds3 subtk is created/updated, its tk.filterObj should be saved to state so it can be recovered from session
+		*/
+		const config = structuredClone(this.state.config)
+		config.subTkFilters = []
+		for (const t of bb.tklst) {
+			if (t.filterObj) {
+				config.subTkFilters.push(t.filterObj)
```

Comment:
We might consider a `global` and `local` filters in the filterObj so users can reuse filter like this in other plots.

Reply:
good idea. can introduce `tk.globalFilter`